extension/test/unit/service/klarna-service.spec.js

```diff
@@ -0,0 +1,98 @@
+const { expect } = require('chai')
+const _ = require('lodash')
+const klarnaService = require('../../../src/service/klarna-service')
+const { KLARNA_DEFAULT_LINE_ITEM_NAME, DEFAULT_PAYMENT_LANGUAGE } = require('../../../src/config/klarna-constants')
+const ctpCart = require('../fixtures/ctp-cart')
+
+describe('klarna-service::createLineItems', () => {
```

Comment:
I guess those tests should be part of the make-payment unit tests.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/294/commits/2124321c5cdf820ced051c2882c217d9f45c1d2c